{"name": "test_add_failure_reason", "status": "passed", "description": "\n        P2 #13: AÃ±adir motivo de fallo\n        ", "start": 1760553691024, "stop": 1760553700507, "uuid": "d635ad00-cd8d-4902-b125-7e336e9a1700", "historyId": "d6fa5ef90c871848e87d0edf946b7c11", "testCaseId": "d6fa5ef90c871848e87d0edf946b7c11", "fullName": "tests.test_rider_functionality.TestRiderFunctionality#test_add_failure_reason", "labels": [{"name": "tag", "value": "test_id('P2-013')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_rider_functionality"}, {"name": "subSuite", "value": "TestRiderFunctionality"}, {"name": "host", "value": "L43429"}, {"name": "thread", "value": "8820-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_rider_functionality"}]}